### DISCUSSION TITLE
Winglogbeat SYN-SENT

### DISCUSSION THREAD

**Discussion by g4l1l30sec** [Security-Onion-Solutions/securityonion#7491] [Unsupported Versions] [answered]
Hello, I have a slight problem with Winlogbeat, I have run a netstat and I see that I have a "SYN_SENT" , it already allows connectivity from "so-allow". I don't know what else to do, I have seen the video that is in the documentation "https://www.youtube.com/watch?v=Xz-7oDrZdQY" and I see that I have everything as is:/, the client is a Windows Server

**Comment by g4l1l30sec** [comment]
Note: The version of Elasticsearch and Winlogbeat are the same.

**Comment by g4l1l30sec** [comment]
Also, this is my config : 





winlogbeat.event_logs:
  - name: Application
    ignore_older: 72h

  - name: System

  - name: Security

  - name: Microsoft-Windows-Sysmon/Operational
  - name: Microsoft-Windows-PowerShell/Operational
    



setup.template.settings:
  index.number_of_shards: 1

setup.kibana:


output.logstash:
  hosts: ["IP_SERVER:5044"]



Note: I know it's not well segmented

**Comment by g4l1l30sec** [comment]
I've been thinking, I have the osquery agent (it works well with Fleet and I also visualize it in kibana).... would this be the problem? ie Osquery and winlogbeat cannot coexist?

**Comment by dougburks** [comment, marked as answer]
It's not necessarily that osquery and winlogbeat cannot coexist. Most folks running osquery don't feel the need to also install winlogbeat.

From https://docs.securityonion.net/en/2.3/host.html:
> You can send logs to Security Onion via your choice of either [osquery](https://docs.securityonion.net/en/2.3/osquery.html#osquery), [Beats](https://docs.securityonion.net/en/2.3/beats.html#beats), [Wazuh](https://docs.securityonion.net/en/2.3/wazuh.html#wazuh), or [Syslog](https://docs.securityonion.net/en/2.3/syslog.html#syslog).

**Reply by g4l1l30sec** [reply]
I already found the "problem", I was testing Security Onion in "EVAL" hehe the beats don't work in that mode. Thanks doug!

**Reply by dougburks** [reply]
For others that may come across this discussion, this is mentioned in the docs at https://docs.securityonion.net/en/2.3/beats.html:
> In order to receive logs from Beats, Security Onion must be running [Logstash](https://docs.securityonion.net/en/2.3/logstash.html#logstash). Evaluation Mode and Import Mode do not run [Logstash](https://docs.securityonion.net/en/2.3/logstash.html#logstash), so you’ll need Standalone or a full Distributed Deployment.
